9/ansible-9.build

```diff
@@ -76,6 +76,7 @@ inspur.ispim: >=2.1.0,<3.0.0
 inspur.sm: >=2.3.0,<3.0.0
 junipernetworks.junos: >=5.3.0,<6.0.0
 kubernetes.core: >=2.4.0,<3.0.0
+kaytus.ksmanage: >=1.2.1,<2.0.0
```

Comment:
I think you need to add this to 10/ansible-10.build as well.

(Also please add this one line up, before kubernetes.core.)

Reply:
thanks! it's not reflected in README in the adding collections to the next major release section

Reply:
Yes, it should be added.